Makefile

```diff
@@ -0,0 +1,17 @@
+VERSION?=v0.0.3-SNAPSHOT
+REGISTRY?=gcr.io
+PROJECT_ID?=kubecost1
+APPNAME?=kube-metrics
+
+release: clean build push clean
+
+build:
+	docker build -f "Dockerfile.metrics" -t ${REGISTRY}/${PROJECT_ID}/${APPNAME}:${VERSION} .
+
+push:
+	docker push ${REGISTRY}/${PROJECT_ID}/${APPNAME}:${VERSION}
+
+clean:
+	docker rm -f ${REGISTRY}/${PROJECT_ID}/${APPNAME}:${VERSION} 2> /dev/null || true
+
+.PHONY: release clean build push
```

Comment:
Yeah, I don't even think I want to commit the Makefile (since I don't even know if our CI uses make, it's just my own choice of build). If this is something we want to do in the future, we can definitely discuss

Reply:
I'm a fan of `make`, for what its worth. CI is fairly primitive (I believe its just running `docker build` and the `Dockerfile` includes unit test-running CI steps) and `make` is ubiquitous so I'd be happy to merge in a Makefile as long as we give it some extra consideration!

Reply:
Looks like the Makefile is gone, so I'll resolve this.